hudi-common/src/main/java/org/apache/hudi/metadata/MetadataPartitionType.java

```diff
@@ -22,27 +22,53 @@
 import java.util.List;
 
 public enum MetadataPartitionType {
-  FILES("files", "files-");
+  // TODO: Make the file group count configurable or auto compute based on the scaling needs
+  FILES(HoodieTableMetadataUtil.PARTITION_NAME_FILES, "files-", 1),
+  COLUMN_STATS(HoodieTableMetadataUtil.PARTITION_NAME_COLUMN_STATS, "col-stats-", 1),
+  BLOOM_FILTERS(HoodieTableMetadataUtil.PARTITION_NAME_BLOOM_FILTERS, "bloom-filters-", 1);
```

Comment:
We need  to make this configurable and make it work for multiple file groups? We need a JIRA for this.

Reply:
metadata partition file groups are now configurable.